app/views/print/list_printers.json.jbuilder

```diff
@@ -0,0 +1,7 @@
+json.array! @printers do |printer|
+  json.name printer.name
+  json.location printer.location
+  json.media printer.media
+  json.weight printer.weight
+  json.duplex printer.duplex
+end
```

Comment:
If you want to you can simplify this file like this: https://github.com/cthit/chalmersit-rails/blob/master/app/views/pages/show.json.jbuilder